runtime.md

```diff
@@ -1,5 +1,47 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information should be stored in `/run/oci/containers`.
+The directory structure for a container is `/run/oci/containers/<containerID>/state.json`.
+By providing a default location that container state is stored external applications can find all containers running on a system.
+
+* **version** (string) Version of the OCI specification used when creating the container.
+* **id** (string) ID is the container's ID.
+* **pid** (int) Pid is the ID of the main process within the container.
+* **root** (string) Root is the path to the container's bundle directory.
```

Comment:
My understanding of today's meeting was that we did need host-side path to the container's root filesystem here. Folks were motivating this entry for mounting additional host-side content into the container's filesystem.  I don't see how the bundle root would help much there.  Maybe bundle-root → `config.json` → `root.path`?  But that assumes that the container wasn't launched with an alternate config, that the config hasn't been tweaked since launch, ….  It seems better to just point folks directly at the container's root filesystem.